src/i18n.ts

```diff
@@ -9,6 +9,9 @@ let numberFormat: Intl.NumberFormat = new Intl.NumberFormat()
 
 export default function i18n(key: string, ...args: Array<string | number>): string {
   let str: string = messages[key] || untranslated[key] || key
+  args.forEach((arg, idx) => {
+    str = str.replace(new RegExp(`%${idx + 1}\\$s`, 'g'), String(arg))
+  })
   args.forEach(a => { str = str.replace('%s', String(a)) })
```

Comment:
It would be nice to check whether this line can be removed. Since we pull new translation from crowdin, all string should have the new format.
If old format strings remain after the merge, I'd suggest to check if they are used in the app.
If we really must keep the old format, then let's keep this replace, but having a single `args.forEach`, not 2 (by tweaking the regex it is probably possible to have a since `str.replace` too).